elastic/integrations
# remove hostname from zookeeper mntr data stream

## Proposed commit message



## Checklist


## Author's Checklist



## How to test this PR locally
Run zookeeper locally
```
docker run -it -d -p 2181:2181 -p 2888:2888 -p 3888:3888 -e ZOO_4LW_COMMANDS_WHITELIST=srvr,mntr,cons,conf zookeeper
```

Add Zookeeper integration in local Kibana
Follow instructions of adding an agent
(to make it work locally I added following lines into my `/etc/hosts` file
```
127.0.0.1       fleet-server
127.0.0.1       elasticsearch
```

For generating some extra metrics I created a new key in zookeeper
```
docker exec -it <ZOOKEEPER_CONTAINER_ID> bash
./bin/zkCli.sh
create /test test-data
get /test
quit
exit
```
Check Kibana Discover page to make sure `zookeeper.mntr.hostname` field is not showing up
<img width="441" alt="Screenshot 2024-11-22 at 2 28 40 PM" src="https://github.com/user-attachments/assets/851702d2-cc79-4a42-81bd-e74fb69164c1">




## Related issues


- Closes #11600

## Screenshots
